tests/test_diff_view.py

```diff
@@ -432,6 +432,46 @@ def setUp(self):
     def tearDown(self):
         unstub()
 
+    @p.expand([
```

Comment:
I'm not to fimiliar with writing these teste what does this do?

Reply:
`@p.expand` = paramterize the tests

`expand` takes a list of tuples/args, and will create a separate test for each args/invocation signature.